modules/central-configuration-options-operator.adoc

```diff
@@ -132,7 +132,7 @@ If no PVC with the given name exists, it will be created. The default value is `
 This is typically used when integrating with services using a private certificate authority.
 
 | `misc.createSCCs`
-| Specify `true` to create SCCs for Central.
+| Specify `true` to create SecurityContextConstraints (SCCs) for Central.
```

Comment:
Per [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting), this should either be:

```
`SecurityContextConstraints` (SCCs)
```
for an "object reference" or
```
security context constraints (SCCs)
```
for a "general reference".